content/en/serverless/installation/nodejs.md

```diff
@@ -159,46 +110,62 @@ npm install datadog-lambda-js
 yarn add datadog-lambda-js
 ```
 
-Then, using the AWS Console or the AWS CLI, add a new `DD_FLUSH_TO_LOG` environment variable set to `true`. This step needs to be repeated for every function you wish to trace.
+See the [latest release][3].
 
-### Instrument your code
+### Configure the Function
 
-Instrument your functions to ingest traces into Datadog:
+1. Set your function's handler to `/opt/nodejs/node_modules/datadog-lambda-js/handler.handler` if using the layer, or `node_modules/datadog-lambda-js/dist/handler.handler` if using the package.
+1. Set the environment variable `DD_LAMBDA_HANDLER` to your original handler, e.g., `myfunc.handler`.
+1. Set environment variable `DD_TRACE_ENABLED` to `true`.
+1. Set environment variable `DD_FLUSH_TO_LOG` to `true`.
+5. Optionally add a `service` and `env` tag with appropriate values to your function.
 
-```
-const { datadog } = require('datadog-lambda-js');
-const tracer = require('dd-trace').init(); // Any manual tracer config goes here.
+### Subscribe the Datadog Forwarder to the Log Groups
 
-// This function will be wrapped in a span
-const longCalculation = tracer.wrap('calculation-long-number', () => {
-    // An expensive calculation goes here
-});
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
 
-// This function will also be wrapped in a span
-module.exports.hello = datadog((event, context, callback) => {
-    longCalculation();
+1. [Install the Datadog Forwarder if you haven't][4].
+2. [Ensure the parameter DdFetchLambdaTags is set to `true`][5].
+3. [Subscribe the Datadog Forwarder to your function's log groups][6].
 
-    callback(null, {
-        statusCode: 200,
-        body: 'Hello from serverless!'
-    });
-});
-```
-
-### Subscribe the Forwarder to log groups
 
-You need the Datadog Forwarder to subscribe to each of your function’s log groups to send traces and enhanced metrics to Datadog.
+[1]: https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html
+[2]: https://github.com/DataDog/datadog-lambda-layer-js/releases
+[3]: https://www.npmjs.com/package/datadog-lambda-js
+[4]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder
+[5]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder/#ddfetchlambdatags
+[6]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#send-aws-service-logs-to-datadog
+{{% /tab %}}
+{{< /tabs >}}
 
-You can quickly verify that you’ve installed the Datadog Forwarder on the [AWS Forwarder page][1] For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][1]. Make sure the Datadog Forwarder is in the same AWS region as the Lambda functions you are monitoring.
+## Quick Start
```

Comment:
```suggestion
## Viewing your data
```

Reply:
Let's keep it as quick start for now, but we should revisit this section and chat about what we want it to look like longer term @apigirl.

Reply:
For now, let's call this "Explore Datadog Serverless Monitoring" in line with what APM and other sections are doing.